.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,44 @@
+---
+name: 🐛 Bug Report
+about: Submit a bug report to help us improve
+labels: "bug"
+---
+
+## 🐛 Bug Report
+
+(A clear and concise description of what the bug is)
+
+### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
```

Comment:
I feel CONTRIBUTING.md may not be so relevant for a bug report . You could skip this section.
```suggestion
## Description 
(A clear and concise description of what the bug is)
```